app/config/Module.scala

```diff
@@ -38,7 +40,8 @@ class Module extends play.api.inject.Module {
       sessionBinding,
       bind[Clock].toInstance(Clock.systemUTC()),
       bind[AppConfig].toSelf,
-      bind[JourneyDataRetrievalAction].to[JourneyDataRetrievalActionImpl].eagerly()
+      bind[JourneyDataRetrievalAction].to[JourneyDataRetrievalActionImpl].eagerly(),
+      bind[SessionDataConnectorConfig].toInstance(ConfigLoaderProvider.get[SessionDataConnectorConfig](configuration))
```

Comment:
Not sure we need this? Should be able to use `ServicesConfig` and load this by calling `.baseUrl` in `AppConfig`?